_posts/0000-01-02-thiagomasanori.md

```diff
@@ -0,0 +1,7 @@
+
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
```

Comment:
It's very easy to make some words **bold** and other words *italic* with Markdown.